butane_core/src/autopk.rs

```diff
@@ -0,0 +1,109 @@
+use super::{FieldType, FromSql, PrimaryKeyType, Result, SqlType, SqlVal, SqlValRef, ToSql};
```

Comment:
optional, but IMO worth considering moving towards https://github.com/Electron100/butane/pull/141 - I've added a comment there to clarify how the two would interact with each other, in case it is relevant.

```diff
diff --git a/butane_core/src/autopk.rs b/butane_core/src/autopk.rs
index 16f3888..9c7d482 100644
--- a/butane_core/src/autopk.rs
+++ b/butane_core/src/autopk.rs
@@ -1,7 +1,9 @@
-use super::{FieldType, FromSql, PrimaryKeyType, Result, SqlType, SqlVal, SqlValRef, ToSql};
-use serde::{Deserialize, Serialize};
 use std::cmp::{Ordering, PartialOrd};
 
+use serde::{Deserialize, Serialize};
+
+use super::{FieldType, FromSql, PrimaryKeyType, Result, SqlType, SqlVal, SqlValRef, ToSql};
+
 #[derive(Clone, Debug, Default, Serialize, Deserialize)]
 pub struct AutoPk<T: PrimaryKeyType> {
     inner: Option<T>,
```

Reply:
Thanks for the update there. I won't take that change *in this PR* but am OK with it going forward in #141. Let's continue the discussion there.